### PR TITLE
docs: add `BlackHole` adapter docs

### PR DESCRIPTION
**Pull request by boesing**
Docs related to laminas/laminas-cache-storage-adapter-blackhole#1

### CODE REVIEW

File: docs/book/storage/adapter.md [hunk 1]
```diff
@@ -561,6 +561,50 @@ Name | Data Type | Default Value | Description
...
+`maxTtl` | 0
+`staticTtl` | `false` or `true`, depending on `psr` option
+`ttlPrecision` | 1
+`useRequestTime` | false
+`lockOnExpire` | 0
+`maxKeyLength` | -1
+`namespaceIsPrefix` | `true`
+`namespaceSeparator` | none
+
+### Adapter specific Options
```
Comment (froschdesign) [inline docs/book/storage/adapter.md]:
```suggestion
### Adapter Specific Options
```

File: docs/book/storage/adapter.md [hunk 2]
```diff
@@ -561,6 +561,50 @@ Name | Data Type | Default Value | Description
...
+
+Name | Data Type | Default Value | Description
+---- | --------- | ------------- | -----------
+`psr` | `bool` |  `false` | Flag to specify whether the adapter should be compatible with `CacheItemPoolDecorator` or `SimpleCacheDecorator`
+
+### Additional notes
+
+The `psr` option was introduce to provide non-BC compatible way to use the `BlackHole` adapter with the PSR-6 and PSR-16 decorator. Ignore this option if this adapter is not used in combination with these decorators. This option is already flagged as internal and thus will be removed in `laminas/laminas-cache-storage-adapter-blackhole` 2.0.
```
Comment (froschdesign) [inline docs/book/storage/adapter.md]: If this is a note of deprecation, then it should be labelled as such.

```suggestion
> ### Deprecation Notice
> 
> The `psr` option was introduce to provide non-BC compatible way to use the `BlackHole` adapter with the PSR-6 and PSR-16 decorator. Ignore this option if this adapter is not used in combination with these decorators. This option is already flagged as internal and thus will be removed in `laminas/laminas-cache-storage-adapter-blackhole` 2.0.
```

File: docs/book/storage/adapter.md [hunk 3]
```diff
@@ -561,6 +561,50 @@ Name | Data Type | Default Value | Description
 ---- | --------- | ------------- | -----------
 `namespace_separator` | `string` |  ":" | A separator for the namespace and prefix.
 
+## The BlackHole Adapter
+
+`Laminas\Cache\Storage\Adapter\BlackHole` **does not** store any cache items.
```
Comment (froschdesign) [inline docs/book/storage/adapter.md]: A description of the possible use should be added here. Like: "This adapter is mostly useful for tests."

**Review by froschdesign — CHANGES_REQUESTED** [review]
I miss a description of the use.

File: docs/book/storage/adapter.md (hunk 1 above)
Comment (boesing) [reply docs/book/storage/adapter.md]: Should I update all other adapter specific options aswell?

Comment (froschdesign) [reply docs/book/storage/adapter.md]: I wanted to spare you that pain because this also means to remove "The" from all headlines. 😉

**Review by froschdesign — APPROVED** [review]

File: docs/book/storage/adapter.md (hunk 1 above)
Comment (boesing) [reply docs/book/storage/adapter.md]: Fixed ;-)

**Pull request merged**